AirCasting/BluetoothSession/BluetoothSessionRecordingController.swift

```diff
@@ -56,33 +57,39 @@ class MobileAirBeamSessionRecordingController: BluetoothSessionRecordingControll
         Resolver.resolve(AirBeamConfigurator.self, args: device)
             .configureMobileSession(location: locationTracker.location.value?.coordinate ?? .undefined,
                                     completion: completion)
+        
         // Info: we're not changing the sessions `status` property here to `.RECORDING` because it is currently
         // being done by the MeasurementStreamStorage class automagically.
         // This is something we might want to change at some point.
+        
+        guard !isRecording else { return } // To make sure we are not recording more than one session at once
```

Comment:
if this scenario (a call to `startRecording`) is unexpected and shouldn't happen maybe we should put an assertion here so that we get a feedback while coding in the future?